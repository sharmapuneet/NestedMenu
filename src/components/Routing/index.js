import React from 'react'
import { Switch, Route } from 'react-router-dom'
import Form from '../Form'
import Message from '../Message'
import Welcome from '../Welcome'

const Routing = () => {
  return (
    <Switch>
      <Route exact path='/form' component={Form}/>
      <Route exact path='/item' component={Message}/>
      <Route exact path='/' component={Welcome}/>
    </Switch>
  )
}

export default Routing
