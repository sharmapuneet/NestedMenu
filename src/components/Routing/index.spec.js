import React from 'react';
import { shallow } from 'enzyme';
import Routing from './index';

describe('Routing', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Routing />)
    expect(wrapper).toMatchSnapshot();
  });
});