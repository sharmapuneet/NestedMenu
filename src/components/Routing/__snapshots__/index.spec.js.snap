// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routing should render correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/form"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/item"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
</Switch>
`;
